crates/physics/src/physx.rs

```diff
@@ -99,6 +99,13 @@ pub fn sync_ecs_physics() -> SystemGroup {
         query((translation().changed(), rotation().changed())).incl(physics_shape());
     let translation_rotation_q2 = translation_rotation_q.query.clone();
 
+    // we need a seperate query here for child entities
+    // Though they have rotation and translation those values may not update
+    // the parents will and local_to_world.changed() will need to be updated
```

Comment:
Hm... it might make more sense to just replace the translation_rotation one, and then maybe have something that reads back changes to ltw to translation rotation.

In general we need to look over the physics system, it was built a long time ago for slightly different requirements that what's needed now

Reply:
How do you feel about merging this as-is and then fixing it up later? The child colliders have been an issue for some time (https://github.com/AmbientRun/Ambient/issues/534), so I think unblocking this and revisiting it later is a decent trade.

Reply:
I think as is it will conflict with the translation_rotation one. @kevzettler if you add `.incl(local_to_parent())` on the query (line 106) it should only affect entities that are children.

Also line 107 it's cloning the wrong query.

But in general; Let's revisit this once the launch is out; I'm not sure how else this will affect things so not sure if anything will break by doing this. We need to set up more test cases for physics also.

Reply:
@FredrikNoren thanks for the code review I have made the fixes in https://github.com/AmbientRun/Ambient/pull/885/commits/9af616c7468bccf29590bbc46f0947ece02d3939

I will look into adding test cases but I do not want to make a large investment if this physics implementation gets rewritten. I will work to keep this branch up todate with main because I will need this child collision behavior for my project.

Reply:
@kevzettler Great! With those fixes I think it actually can't affect anything else and it looks good to me so I'll go ahead and merge it